pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -72,8 +72,13 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
-        seccompProfile:
-          type: RuntimeDefault
+        # TODO(user): For common cases that do not require escalating privileges
+        # it is recommended ensure that all your Pods/Containers are restrictive.
+        # On this case, you can uncomment the following code. However, if you are looking 
+        # for to built projects which must work on old Kubernetes versions < 1.19 or 
+        # on vendors versions which are NOT supporting this field (i.e. Openshift < 4.11 ).
```

Comment:
Some wording tweaks:
```suggestion
        # TODO(user): For common cases that do not require escalating privileges
        # it is recommended to ensure that all your Pods/Containers are restrictive.
        # In this case, you can uncomment the following code. If your project must 
        # work on old Kubernetes versions < 1.19 or on vendor versions which
        #  do NOT support this field (i.e. Openshift < 4.11 ), leave this commented out.
```